pkg/registry/images.go

```diff
@@ -215,6 +215,17 @@ func deleteUnusedImages(ctx context.Context, registry types.RegistrySettings, us
 	digestsInRegistry := map[string]string{}
 	for _, r := range searchResult {
 		imageName := path.Join(registry.Hostname, r.Name)
+
+		parts := strings.Split(imageName, "/")
```

Comment:
Why not use `reference.Path(ref.DockerReference())`?

Reply:
cuz it's not parsed as a reference yet at that point. i guess i could, but we do use this pattern in our codebase